ytyp/properties.py

```diff
@@ -123,6 +123,10 @@ def get_room_to_name(self):
     def get_name(self):
         return f"{self.room_from_name} to {self.room_to_name}"
 
+    def update_audio_occlusion(self, context):
+        if self.audio_occlusion < 0:
+            self.audio_occlusion = ((2**31) - 1)
```

Comment:
is it intended for it to be the max value for a 32-bit *signed* integer? If not, you can do `2**32 - 1` for max value of an unsigned integer.

Reply:
if you also want some sort of rollover effect like overflowing/underflowing usually does, you could also do:

```py
if self.audio_occlusion < 0:
    self.audio_occlusion = self.audio_occlusion % 4294967296L # const value of 2 ** 32
```

Reply:
@CFox04 so does this fix it or?

Reply:
> is it intended for it to be the max value for a 32-bit _signed_ integer? If not, you can do `2**32 - 1` for max value of an unsigned integer.

No that was not intended. I'll fix this.

Reply:
So the ``bpy.props.IntProperty`` does not seem to let me store a 32-bit unsigned integer even if I have the ``subtype="UNSIGNED"`` parameter. I just made it a string property and ensured it is the correct value in the update function.